app/views/admin/ambassadors/show.html.haml

```diff
@@ -0,0 +1,30 @@
+%h1= "Ambassador: #{@ambassador.username}"
+
+- if @ambassador.avatar
+  - unless @ambassador.avatar.url(:medium) == 'https://files.bikeindex.org/blank.png'
+    = image_tag @ambassador.avatar.url(:medium), class: 'users-show-avatar'
+
+
+%table.table.table-striped.table-hover.table-bordered.table-sm.without-exterior-border
```

Comment:
If you use `table.table-list` without any extra classes, it will make a definition list like the other definition table lists in admin, and you don't need to bold the first cell